how to use with webpack2?

```
var onMutate = require('jquery-onmutate');
var jQuery = require('jquery');
onMutate(jQuery);
```

getting

`Module not found: Error: Can't resolve 'jquery-onmutate' in 'E:\Coding\buharmetre\assets\js\common'`

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "jquery-onmutate",
   "version": "1.4.2",
   "description": "A MutationObserver wrapper for jQuery, allowing you to listen for element creation, attribute modification, and text changes.",
-  "main": "jquery.onmutate.js",
+  "main": "src/jquery.onmutate.js",
   "directories": {
     "example": "example"
   },

